src/pod.ts

```diff
@@ -423,6 +442,12 @@ export interface AbstractPodProps extends base.ResourceProps {
    * @default false
    */
   readonly isolate?: boolean;
+  /**
+   * Creates readinessGate for the ingress management
+   *
+   * @default k8s.PodReadinessGate[]
+   */
+  readonly readinessGates?: k8s.PodReadinessGate[];
```

Comment:
We don't expose `k8s.*` interfaces or resource from the L2 layer, as this creates a coupling between the L1 and L2 API's, which is undesired. 

Instead. create an interface in this file called `PodRedinessGate` which includes whatever properties you want to expose. We prefer some duplication over creating the aforementioned coupling. 

Also, looking at the available properties on this struct, I see:

```ts
  /**
   * ConditionType refers to a condition in the pod's condition list with matching type.
   *
   * @schema io.k8s.api.core.v1.PodReadinessGate#conditionType
   */
  readonly conditionType: string;
```

What are the values that `conditionType` can accept? Are these really arbitrary strings or can we wrap it with a type safe API? This kind of stuff is exactly where cdk8s-plus shines and can provide a much simpler experience over just using the L1 layer (or YAML for that matter).